### PR TITLE
Upgrade graphQL v11

### PR DESCRIPTION
**Pull request by GP4cK**
Closes https://github.com/TriPSs/nestjs-query/issues/109
Closes https://github.com/TriPSs/nestjs-query/pull/111

In the first commit, I followed the steps in the [migration guide](https://docs.nestjs.com/graphql/migration-guide) and regenerated the snapshots (see the extra `!`).
In the second commit, I just removed so unused/deprecated packages.

### CODE REVIEW

**Comment by GP4cK** [comment]
For the e2e tests, the http status changed from 400 to 500. I believe the error comes from this file `examples/complexity/src/complexity.plugin.ts`. If we wanted to keep 400, I think we could throw a `BadRequestException` instead.

File: examples/filters/e2e/todo-item.resolver.spec.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('TodoItemResolver (filters - e2e)', () => {
         })
         .expect(200)
         .then(({ body }) => {
-          expect(body.errors[0].extensions.response.message[0]).toBe('filter.There was no filter provided for "completed"!')
+          expect(body.errors[0].extensions.originalError.message[0]).toBe('filter.There was no filter provided for "completed"!')
```
Comment (TriPSs) [inline examples/filters/e2e/todo-item.resolver.spec.ts:100]: Would this not be considered a breaking change?

**Review by TriPSs — COMMENTED** [review]
Thanks again for your help!

File: examples/filters/e2e/todo-item.resolver.spec.ts (hunk 1 above)
Comment (GP4cK) [reply examples/filters/e2e/todo-item.resolver.spec.ts:100]: Yes, this happened between nestjs/graphql 10 and 11: https://github.com/nestjs/graphql/compare/v10.2.1...v11.0.4#diff-c3a3b52cf11536118537eaffe950a721dd8246bde13d7681c96678d85bec445fR248
If you look in `packages/apollo/lib/drivers/apollo-base.driver.ts`, in line 248 in the new file, they changed from `response` to `originalError`

Comment (GP4cK) [reply examples/filters/e2e/todo-item.resolver.spec.ts:100]: Once this is merged, it's probably best if the new version that you will publish is v3.0.0

Comment (TriPSs) [reply examples/filters/e2e/todo-item.resolver.spec.ts:100]: Okay thanks, then I think it's also best to combine this with #31.

Comment (GP4cK) [reply examples/filters/e2e/todo-item.resolver.spec.ts:100]: Sure. Do you need help to fix the CI? It's kinda the end of day for me and from Friday, I'll be on leave for a few days but I can try to have a look.

Comment (TriPSs) [reply examples/filters/e2e/todo-item.resolver.spec.ts:100]: Yea would be awesome! I wanted to check it out portably this Friday so if it's still open I can finish it up than :)

**Pull request merged**